src/main/kotlin/no/nav/syfo/narmestelederrelasjon/NarmesteLederRelasjonService.kt

```diff
@@ -5,20 +5,21 @@ import no.nav.syfo.client.pdl.PdlClient
 import no.nav.syfo.domain.PersonIdentNumber
 import no.nav.syfo.narmestelederrelasjon.database.domain.toNarmesteLederRelasjonList
 import no.nav.syfo.narmestelederrelasjon.database.getNarmesteLederRelasjonList
-import no.nav.syfo.narmestelederrelasjon.domain.NarmesteLederRelasjon
-import no.nav.syfo.narmestelederrelasjon.domain.addNarmesteLederName
+import no.nav.syfo.narmestelederrelasjon.domain.*
 
 class NarmesteLederRelasjonService(
     private val database: DatabaseInterface,
     private val pdlClient: PdlClient,
 ) {
     suspend fun getNarmestelederRelasjonList(
         callId: String,
-        arbeidstakerPersonIdentNumber: PersonIdentNumber,
+        personIdentNumber: PersonIdentNumber,
+        shouldGetAnsatte: Boolean,
     ): List<NarmesteLederRelasjon> {
```

Comment:
I stedet for ha delte funksjoner for disse to "løpene" og sende nedover `shouldGetAnsatte`, så synes jeg det er ryddigere å holde de separert.
Altså et løp for getNarmestelederRelasjonList(arbeidstakerPersonIdentNumber) for AT og et for getNarmestelederRelasjonList(narmeste_leder_personident) for NL.

Reply:
Ja, det er vel kanskje lurt å dele opp, men da blir det også dobbelt så mye kode, hvor 98% er duplikat 🤷 Jeg prøver å dele i morgen 👍

Reply:
Blir flere linjer kode ja, men tror det blir lettere å lese/vedlikeholde da :)